### PR TITLE
feat: Adds a simple watcher to auto rebuild on each js change

### PR DESCRIPTION
**Pull request by jlengrand**
EDIT: It also reloads the extension on each rebuild now <3

Helps a lot when working on the extension

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
   "description": "",
   "main": "content_script.js",
   "scripts": {
+    "reload": "open http://reload.extensions/",
+    "dev": "chokidar \"src/*.js\" -c \"npm run build;npm run reload\"\n\n",
```
Comment (thepassle) [inline package.json]: Can we change this to `start`?

File: README.md [hunk 2]
```diff
@@ -19,7 +20,16 @@ Chrome extension to find custom elements on a page, and search the catalog for e
...
-- Go to <a href="chrome://extensions/">chrome://extensions/</a>
+- Go to [chrome://extensions/](chrome://extensions/)
 - Toggle 'Developer mode' in top right corner
 - Click 'Load unpacked' in top left corner
 - Select the root folder of this project
+
+If you are implementing and want to test repeatedly:
+
+- Install [Extensions Reloader](https://chrome.google.com/webstore/detail/extensions-reloader/fimgfedafeadlieiabdeeaodndnlbhid)
+- Running `npm run dev` will rebuild on each change in the `src` folder, and trigger a reload of the extension.
```
Comment (thepassle) [inline README.md]: lets change to `npm start`

**Comment by thepassle** [comment]
very nice 🙂

File: package.json (hunk 1 above)
Comment (jlengrand) [reply package.json]: Sure thing. That's what I had at first but I thought start had some kind of server feel to it.

**Comment by jlengrand** [comment]
Fixed all and closed in favor of #8

**Pull request closed without merging**